src/freertos_drivers/common/MCAN.hxx

```diff
@@ -24,15 +24,18 @@
  * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
  * POSSIBILITY OF SUCH DAMAGE.
  *
- * @file TCAN4550Can.hxx
- * This file implements the CAN driver for the TCAN4550 CAN Controller.
+ * @file MCAN.hxx
+ * This file implements a generic CAN driver for the MCAN controller IP.
```

Comment:
Maybe it is a good idea to summarize here what should eventually be moved out of this file related to TCAN4550 and/or STM32MCAN.